src/reducers/filter.js

```diff
@@ -0,0 +1,11 @@
+const filterReducer = (state = [], action) => {
+  if (action.type === 'CHANGE_FILTER') {
+    return {
+      ...state,
+      filter: action.payload,
+    };
+  }
+  return state;
+};
```

Comment:
[Optional] Since `action` is not used **directly** anywhere else in the function, I would [`destructure`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) the properties I want to use instead.  This way, I can use `payload` and `type` directly in the function instead of `action.payload` and `action.type` respectively. What do you think? 🤔